### DISCUSSION TITLE
4/23(일) 실제 디바이스가 없을 경우 개발 환경에서 할 수 있는 것과 없는 것을 설명하시오.

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#99] [General]
swift 시뮬레이터에 관한 질문 입니다. 나중에 개발을 하다가 코드를 짰는데 시뮬레이터가 강제로 꺼질 수 있습니다. 그럴 떄 나의 코드를 의심하지 말고 알아두면 잘 응용할 수 있을 것 입니다. 😈

**Comment by Eunho0922** [comment]
넵
